### ISSUE TITLE
Refactor file names, variable names, etc.

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/optmeowt-browser-extension#172]
In the general sprit of refactoring, we realized today that the naming convention for files is not uniform (the same may be true for variables and other names). I am not particular about how to go about this, but it should be uniformly camel case, with underscore, with hyphen, ...

**Comment by kbeliauski** [comment]
Will do that once @kalicki1 is done refactoring the whole codebase

**Comment by kalicki1** [comment]
I have been trying to rename variables and functions when I came across them during the refactor. We can do one big last scan over the codebase prior to our release to see if there's anything else we want to check and then move forward with closing this issue.

**Comment by kalicki1** [comment]
One thought I had was to rename a few of the modules that get bundled into `background.bundle.js` as the extension background script. Currently the hierarchy of function calls (within the bundled code and source code) is as follows: 

`background.js` <-- `listeners-(chrome/firefox).js` <-- `events.js`

Where 
- `background.js` holds: enable()/disable() and calls to browser specific listeners in `listeners-chrome.js`/`listeners-firefox.js`, as well as initializes the default extension settings
- `listeners-chrome.js`/`listeners-firefox.js` holds all the listeners that pause requests to add our Do-Not-Sell signals, which call functions from `events.js`
- `events.js` holds all the functions that get called by the listeners one level above it, as well as all of our cached variables. This means `events.js` also must handle our message handling to and from the background script. 

Here are my suggestions. Let me know what makes sense.

1) One thought I had was to rename `background.js` as `init.js`. This is simply because none of the persistent functionality is held here, it simply initializes the default settings and flips the enable() function, triggering the listeners and on install functionality 
2) Either rename `events.js` as `background.js`, OR 
3) Split `events.js` into two new modules. These two would be along the lines of a module that doesn't do anything, just holds the functions that are called by the background, which we can keep named `events.js`, and another that holds all of the cached values and handles the message passing, effectively making it the "background" to the rest of the extension. This could then be renamed `background.js` since it's the one that would interface with the storage, content scripts, and frontend. It could also be less confusing to instead name this message passing/cache module as `messages.js` or something similar. 

This would look like: 

`init.js` <-- `listeners-(chrome/firefox).js` <-- `effective background`/`message handling` <-- `functions module`

These are just a few long-winded thoughts I had about the structure of the background page source code for readability and for understanding for how the project is structured for new members. Any thoughts are appreciated!

**Comment by SebastianZimmeck** [comment]
All of us will look a bit into this for next, think about this, and comment here. background.js for @kalicki1 that there is something on-going. And @stanleymarkman is saying we should just name the files according to what they do.

**Comment by stanleymarkman** [comment]
Just some initial thoughts-
Background.js sets up listeners- so setup_listeners.js?
Events.js isn't super descriptive, but handles both web traffic and other browser events. Either rename it browser_events.js, or split it into handle_webtraffic and handle_browserevents

**Comment by SebastianZimmeck** [comment]
> Background.js sets up listeners- so setup_listeners.js?
> browser_events.js, or split it into handle_webtraffic and handle_browserevents

I like @stanleymarkman's points.

Also, as @kalicki1 mentioned during our call today, for me a background.js script listens for events the whole time the extension is running. As the current background.js does not do that init.js or setup_listeners.js may be a better name, indeed.

**Comment by SebastianZimmeck** [comment]
OK, I took a very quick glance at this, and for me this is good to go for the release.

**Comment by kalicki1** [comment]
I think I agree with @SebastianZimmeck about this, unless @stanleymarkman you have strong feelings about renaming our background scripts like you mentioned above. I haven't pushed any file name changes recently.

**Comment by SebastianZimmeck** [comment]
I am closing this as we all seem to be OK with how it stands. Feel free to reopen everyone as necessary.
